page-objects/pages/DashboardPage.js

```diff
@@ -1 +1,53 @@
-/*[url/dashboard]*/
\ No newline at end of file
+/*[url/dashboard]*/
+'use strict';
+
+const { expect } = require('chai');
```

Comment:
what is chai?

Reply:
chai is a test framework

Reply:
we don't need "chai" import there, we have access to this globally (without any imports). It was removed